src/main/java/com/flansmod/common/guns/EntityBullet.java

```diff
@@ -1491,4 +1531,210 @@ public void readSpawnData(ByteBuf data) {
     public boolean isBurning() {
         return false;
     }
+    
+    private ArrayList<MovingObjectPosition> rayTraceAllBlocks(World world, Vec3 p_147447_1_, Vec3 p_147447_2_, boolean p_147447_3_, boolean p_147447_4_, boolean p_147447_5_)
```

Comment:
Would be nice if these obfuscated names were renamed to something more readable.

Reply:
I agree, it's copied from the vanilla rayTraceBlocks with a couple modifications